environment/deployments/science-platform/variables.tf

```diff
@@ -192,12 +192,12 @@ variable "node_pool_1_initial_node_count" {
   default = 1
 }
 
-# variable "node_pools" {
-#   type        = list(map(string))
-#   description = "List of maps containing node pools"
+variable "node_pools" {
+  type        = list(map(string))
+  description = "List of maps containing node pools"
 
-#   default = [{}]
-# }
+  default = [{}]
```

Comment:
This default is fine, but I don't see where the node pools are actually set

Reply:
They are defined in env/integration.tfvars (which is unchanged)